core/src/test/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionFixture.scala

```diff
@@ -0,0 +1,82 @@
+package org.broadinstitute.dsde.rawls.expressions
+
+// test data for Expression Validation and Parsing
+
+trait ExpressionFixture {
+  // Inputs which will not validate, so they don't need to be handled by the parser
+  // ""
```

Comment:
i thought this was a comment relating to `parseableInputExpressions` at first. maybe move this closer to `unparseableInputExpressions` and rephrase to say something like "NOTE: empty strings will be rejected in the validation stage, so we don't test them here".

same for the empty-string outputs case below